examples/view_3d_mns_map.html

```diff
@@ -72,9 +72,8 @@
                     new itowns.ElevationLayer(config.id, config),
                 ).then(debugMenu.addLayerGUI.bind(debugMenu));
             }
-
-            itowns.Fetcher.json('./layers/JSONLayers/IGN_MNT.json').then(addElevationLayerFromConfig);
             itowns.Fetcher.json('./layers/JSONLayers/IGN_MNS_HIGHRES.json').then(addElevationLayerFromConfig);
+            // itowns.Fetcher.json('./layers/JSONLayers/IGN_MNT.json').then(addElevationLayerFromConfig);
```

Comment:
There is a probleme with the dataset linked with this configuration file or with the configuration file (IGN_MNT.json), that appear with the new changes. As it is only used in this example and the extra Elevation layer doesn't really seems to be important. I prefered to removed it and it can be eluded when the issue #2000 will be studied.

Reply:
Ok, could you describe this problem in #2000 please so we don't forget about it ?